website/docs/r/cloudfunctions_function.html.markdown

```diff
@@ -109,7 +109,7 @@ The following arguments are supported:
 * `name` - (Required) A user-defined name of the function. Function names must be unique globally.
 
 * `runtime` - (Required) The runtime in which the function is going to run.
-Eg. `"nodejs8"`, `"nodejs10"`, `"nodejs12"`, `"python37"`, `"python38"`,`"go111"`, `"go113"`.
+Eg. `"nodejs10"`, `"nodejs12"`, `"nodejs14"`, `"python37"`, `"python38"`, `"python39"`, `"go111"`, `"go113"`, `"java11"`, `"ruby26"`. Check the [official doc](https://cloud.google.com/functions/docs/concepts/exec#runtimes) for the up-to-date list.
```

Comment:
I don't see `python39` in the list of available runtimes linked here. Is there somewhere else I should be looking?

It might also be worth leaving out go111 since it's already deprecated; interested users can check the full list.